Port 7819 k 8 s visualization (#1135)
port-labs/port-docs: docs/guides-and-tutorials/visualize-service-k8s-runtime.md

### Define the connection between services and workloads

Now that we have our <PortTooltip id="blueprint">blueprints</PortTooltip> set up, we want to model the logical connection between them by relating the `Workload` blueprint to the `Service` blueprint. This will grant us some helpful context in our Software catalog, allowing us to see relevant `Workloads` in a `Service`'s context, or a `Service`'s property directly in its corresponding `Workload`.

> In this guide we will create one relation named `service` which will represent the different services the workloads are running.

Reviewer: ```suggestion
In this guide we will create one relation named `service` which will represent the service that a workload is running.
```
